Unix/configure

```diff
@@ -166,9 +166,10 @@ do
       enable_werror=1
       ;;
 
-    --enable-wchar)
-      enable_wchar=1
-      ;;
+## Disabling support for wchar builds
+##    --enable-wchar)
+##      enable_wchar=1
+##      ;;
```

Comment:
I guess I didn't want to purge everything to do with wchar from both the files as it was a bigger change. I will re-evaluate.